db/schema.sql

```diff
@@ -44,10 +44,18 @@ CREATE TABLE activities
 CREATE TABLE visits
 (
   id SERIAL PRIMARY KEY,
-  usersId INTEGER REFERENCES users,
+  usersId INTEGER REFERENCES users(id),
   activitiesId INTEGER REFERENCES activities(id),
   cb_id INTEGER REFERENCES cbusiness(id),
   date TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
+CREATE TABLE feedback
+(
+  id SERIAL PRIMARY KEY,
+  cb_id INTEGER REFERENCES cbusiness(id),
+  user_feedback INTEGER NOT NULL,
```

Comment:
🙏 please can we destroy as many references to "user" as possible, IMO it's nothing but confusing. IMO `feedback_score` is better because we are collecting quantitative feedback, not qualitative.

Reply:
Yeah, makes sense 😊